pytensor/tensor/blas.py

```diff
@@ -113,23 +113,22 @@
 _logger = logging.getLogger("pytensor.tensor.blas")
 
 
-# If check_init_y() == True we need to initialize y when beta == 0.
-def check_init_y():
-    # TODO: What is going on here?
+def gemv_impl_sensitive_to_unitialized_y():
```

Comment:
Do you mean "initialized" ?

Even if you do, I don't understand what an "initialized" y is.

Reply:
So is "unitialized" a typo or not?

Reply:
I don't know is it? I thought it was a word that means not initialized

Reply:
Wouldn't that be "uninitialized"? 

"unitialized" looks like "not italicized" to me

Reply:
Seems to be what you're saying: https://en.wikipedia.org/wiki/Uninitialized_variable

nini it is

Reply:
Changed names again

Reply:
I still see the old names, are you wanting to keep them?

Reply:
Github is stuck for some reason: 
![image](https://github.com/user-attachments/assets/f1d7c967-9c81-41d6-9c83-d5b56f1a87e7)

See the processing updates

Reply:
Ah ok. Well I approved so you can merge it when you're convinced it's right